README.md

```diff
@@ -33,89 +33,92 @@ System.Web.Bem - БЭМ-инфрастурктура для ASP.NET MVC.
 
-Бандлы технологий имеют имя `<bundle_name>.<tech_ext>` и сохраняются в папку, где находится декларация. Например, файл шаблонов (bemhtml.js) для бандла `default` будет иметь путь `/Bem/desktop.bundles/default/default.bemhtml.js`. Таким образом, после сбоки проекта вы будете наблюдать примерно такую картину:
 ```
 <Project root>
 ├─ Bem
 │  ├─ ...
 │  └─ desktop.bundles
 │     ├─ default 
-│     │  ├─ default.bemdecl.js  // декларация бандла
-│     │  ├─ default.bemhtml.js  // бандл с шаблонами 
-│     │  ├─ default.js          // бандл с клиентским кодом 
-│     │  └─ default.css         // бандл со стилями 
-│     │  └─ ...                 // бандлы других технологий
+│     │  ├─ default.bemdecl.js  // bundle declaration
+│     │  ├─ default.bemhtml.js  // templates bundle 
+│     │  ├─ default.js          // client-side code bundle 
+│     │  └─ default.css         // styles bundle 
+│     │  └─ ...                 // bundles of other technologies
 │     └─ ...
 └─ ...
 ```
-Во время работы приложения бандл шаблонов используется для формирования html (на сервере и клиенте), бандлы js и css подключаются на страницы и используются на клиенте. Во время серверной шаблонизации будет использоваться бандл с шаблонами, указанный в настройках приложения. Бандлы js и css нужно самостоятельно подключить на страницу.
 
-### Серверная шаблонизация BEMHTML
+During the application is running the bundle of templates is used for html generation (on the server and the client side), js bundles and style bundles are included in the pages and used on client side. During the server-side html generation will be used the templates bundle which is selected by application settings. Client-side bundles (js and css) should be included to the page manually.
+
+### Server side BEMHTML templates
 
-[BEMHTML](https://github.com/bem/bem-xjst) - специальный шаблонизатор, который удобно использовать в БЭМ-проектах.
+[BEMHTML](https://github.com/bem/bem-xjst) is the special templates engine which is convenient for BEM-projects.
 
-При установке NuGet пакета System.Web.Bem в проект будет добавлена .NET библиотека для шаблонизации BEMHTML шаблонов на стороне сервера во время работы приложения. Чтобы передать на клиент страницу, сформированную с помощью bemhtml шаблонов, просто верните из контроллера экземпляр класса `BemhtmlResult`, передав ему в конструктор нужный bemjson.
+During the installation of [System.Web.Bem package](https://www.nuget.org/packages/System.Web.Bem) NuGet package into the project the reference for .NET library named System.Web.Bem will be added (this library allows to use BEMHTML templates on the server side). To generate a html-page using BEMHTML on the server-side you should just return the instance of the `BemhtmlResult` class from the controller's method. Send the essential *bemjson* to constructor of the `BemhtmlResult`.
```

Comment:
When the NuGet [System.Web.Bem package](https://www.nuget.org/packages/System.Web.Bem) is installed in the project, a reference to the System.Web.Bem .NET library is added in order to use BEMHTML templates on the server side. To generate an HTML page using BEMHTML on the server, just return an instance of the `BemhtmlResult` class from the controller's method and send the necessary *bemjson* to the `BemhtmlResult` constructor.